packages/next/src/builders/build/build.impl.ts

```diff
@@ -4,12 +4,10 @@ import {
   createBuilder
 } from '@angular-devkit/architect';
 import { JsonObject } from '@angular-devkit/core';
-import { PHASE_PRODUCTION_BUILD } from 'next/dist/next-server/lib/constants';
 import build from 'next/dist/build';
 import * as path from 'path';
 import { from, Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
-import { prepareConfig } from '../../utils/config';
 
 try {
   require('dotenv').config();
```

Comment:
Just wondering if we need to keep this in here? Isn't it more typical for this to be required in the config file itself? If so, would folks find it confusing not to see it in the config file?